### ISSUE TITLE
DefaultKafkaProducerFactory logging out of place ?

### ISSUE THREAD

**Issue by jorgheymans** [spring-projects/spring-kafka#1270]
In 2.3.0 , `DefaultKafkaProducerFactory` logs this:

```
...
		String txId = (String) this.configs.get(ProducerConfig.TRANSACTIONAL_ID_CONFIG);
		if (StringUtils.hasText(txId)) {
			setTransactionIdPrefix(txId);
			LOGGER.info(() -> "If 'setTransactionIdPrefix()' is not going to be configured, "
					+ "the existing 'transactional.id' config with value: '" + txId
					+ "' will be suffixed for concurrent transactions support.");
		}
...
```

this seems a bit out of place, especially since i'm using `setTransactionIdPrefix()` (but at the time of constructor there is no way of knowing) and i am seeing this at INFO level.  Also the wording is strange.

**Comment by artembilan** [comment]
Yeah... I think we should move it to the `afterPropertiesSet()` with an appropriate `if...else` and with rewording.

Feel free to raise a Pull Request and we will take a look closer!

Thank you for heads up!

**Comment by jorgheymans** [comment]
this was fixed in #1281
